File: MainActivity.java
package com.example.budget_min;



import androidx.appcompat.app.AppCompatActivity;

import android.os.Bundle;
import android.view.View;
import android.widget.AdapterView;
import android.widget.ArrayAdapter;
import android.widget.Spinner;
import android.widget.Toast;

import java.util.ArrayList;
import java.util.List;

public class MainActivity extends AppCompatActivity {
    Spinner dropdownmenu;

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);

        dropdownmenu = (Spinner) findViewById(R.id.spinner4);
        List<String> list = new ArrayList<>();
        list.add("Rent");
        list.add("Water Bill");
        list.add("Internet Bill");
        list.add("Electric Bill");
        list.add("Coffee");
        list.add("Movie");
        list.add("UBER");
        list.add("OTHER");

        ArrayAdapter<String> adapter = new ArrayAdapter<String>(this, android.R.layout.simple_spinner_item, list);


        adapter.setDropDownViewResource(android.R.layout.simple_spinner_dropdown_item);

        dropdownmenu.setAdapter(adapter);

        dropdownmenu.setOnItemSelectedListener(new AdapterView.OnItemSelectedListener() {
            @Override
            public void onItemSelected(AdapterView<?> parent, View view, int position, long id) {
                String Ivalue=parent.getItemAtPosition(position).toString();
                Toast.makeText(MainActivity.this, "Selection "+Ivalue,Toast.LENGTH_SHORT).show();
            }


            @Override
            public void onNothingSelected(AdapterView<?> parent) {

            }
        });
    }
}

File: rahin10-8/MainActivity.kt
package com.example.budgetminimalism

import android.content.Intent
import android.os.Bundle
import android.util.Log.d
import androidx.appcompat.app.AppCompatActivity
import kotlinx.android.synthetic.main.activity_about_view.*
import kotlinx.android.synthetic.main.activity_main.*
import kotlinx.android.synthetic.main.content_main.*

class MainActivity : AppCompatActivity() {

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)
        setSupportActionBar(toolbar)

        ViewButton.setOnClickListener {
            d("rain","button was pressed")
            startActivity(Intent(this,AboutView::class.java))
        }
        RecentActivityButton.setOnClickListener {
            d("rain","button 2 was pressed")
            startActivity(Intent(this,AboutRecentActivity::class.java))
        }
    }
}
